mobile/status.go

```diff
@@ -1356,3 +1405,45 @@ func GetRandomMnemonic() string {
 	}
 	return mnemonic
 }
+
+func ToggleCentralizedMetrics(requestJSON string) string {
+	var request requests.ToggleCentralizedMetrics
+	err := json.Unmarshal([]byte(requestJSON), &request)
+	if err != nil {
+		return makeJSONResponse(err)
+	}
+
+	err = request.Validate()
+	if err != nil {
+		return makeJSONResponse(err)
+	}
+
+	err = statusBackend.ToggleCentralizedMetrics(request.Enabled)
+	if err != nil {
+		return makeJSONResponse(err)
+	}
+
+	return ""
+}
+
+func AddCentralizedMetrics(requestJSON string) string {
```

Comment:
It would be clearer in the singular form because only one metric can be added by this function.